infrastructure/server/network_security_groups.tf

```diff
@@ -0,0 +1,121 @@
+resource "aws_security_group" "fam_app_sg" {
+    name = "fam_app_sg"
+    description = "FAM custom security group for application tier (lambdas)."
+    vpc_id = data.aws_vpc.selected.id
+    revoke_rules_on_delete = true
+
+    tags = {
+        Name = "fam_app_sg"
+        managed-by = "terraform"
+    }
+
+    ingress {
+        from_port = 0
+        to_port = 0
+        protocol = "-1"
+        cidr_blocks = ["10.10.32.0/20", "10.10.128.0/20"]
```

Comment:
just for curious Basil @basilv, I wonder how we know which ips we need to block? are these known bad ips?